src/Symfony/Component/Console/Application.php

```diff
@@ -833,7 +833,7 @@ public function renderThrowable(\Throwable $e, OutputInterface $output): void
     private function finishRenderThrowableOrException(OutputInterface $output): void
     {
         if (null !== $this->runningCommand) {
-            $output->writeln(sprintf('<info>%s</info>', sprintf($this->runningCommand->getSynopsis(), $this->getName())), OutputInterface::VERBOSITY_QUIET);
+            $output->writeln(sprintf('<info>%s</info>', OutputFormatter::escape(sprintf($this->runningCommand->getSynopsis(), $this->getName()))), OutputInterface::VERBOSITY_QUIET);
```

Comment:
Not sure how I can add a test case for that. @chalasr Do you have an idea?

Reply:
I think you can borrow one from ApplicationTest e.g. https://github.com/symfony/symfony/blob/14540c1ae54806e1c3b0b23934776ce3733d76b1/src/Symfony/Component/Console/Tests/ApplicationTest.php#L853-L866

Reply:
(let me know if you are stuck at reproducing this specific case :))

Reply:
Thanks! What do you think? 6ee41c658ab9a7175ce8bf9469fc7b085db5bf22